charts/camunda-platform/templates/zeebe/statefulset.yaml

```diff
@@ -85,6 +85,35 @@ spec:
               value: {{ .Values.zeebe.ioThreadCount  | quote }}
             - name: ZEEBE_BROKER_GATEWAY_ENABLE
               value: "false"
+            {{- if .Values.global.elasticsearch.external  }}
+              {{- if .Values.global.elasticsearch.openSearch}}
+            - name: ZEEBE_BROKER_EXPORTERS_OPENSEARCH_ARGS_AUTHENTICATION_USERNAME
+              value: {{ .Values.global.elasticsearch.auth.username | quote }}
+            - name: ZEEBE_BROKER_EXPORTERS_OPENSEARCH_ARGS_AUTHENTICATION_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: elastic-password
+                  key: password
+            - name: ZEEBE_BROKER_EXPORTERS_OPENSEARCH_CLASSNAME
+              value: "io.camunda.zeebe.exporter.opensearch.OpensearchExporter"
+            - name: ZEEBE_BROKER_EXPORTERS_OPENSEARCH_ARGS_URL
+              value: {{ include "camundaPlatform.elasticsearchURL" . | quote }}
+            - name: ZEEBE_BROKER_EXPORTERS_OPENSEARCH_ARGS_BULK_SIZE
+              value: "1"
+            - name: ZEEBE_BROKER_EXPORTERS_OPENSEARCH_ARGS_INDEX_PROCESSMESSAGESUBSCRIPTION
+              value: "true"
+              {{- else}}
+            - name: ZEEBE_BROKER_EXPORTERS_ELASTICSEARCH_ARGS_AUTHENTICATION_USERNAME
+              value: {{ .Values.global.elasticsearch.auth.username | quote }}
+            - name: ZEEBE_BROKER_EXPORTERS_ELASTICSEARCH_ARGS_AUTHENTICATION_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: elastic-password
+                  key: password
+              {{- end}}
+            {{- end }}
+            # {{- if and .Values.global.elasticsearch.external (not .Values.global.elasticsearch.openSearch)}}
+            # {{- end }}
             {{- if not .Values.global.elasticsearch.disableExporter }}
             - name: ZEEBE_BROKER_EXPORTERS_ELASTICSEARCH_CLASSNAME
```

Comment:
❌ this means the elastic exporter also runs if you use opensearch right? these elasitc properties should only be used if you are connecting to elasticsearch. this relates to the previous comment where we suggested to have dedicated config sections for both. So you only configure either of them.

Reply:
I will see if this option can be enabled by default by modifying the template file so the user does not have to set the `disableExporter` option when enabling opensearch

I will also talk to my team about separating the configs.

Reply:
> The user would have to explicitly disable elasticsearch exporter from the values.yaml like this:
> 
> ```yaml
> global:
>   elasticsearch:
>     disableExporter: true
>     external: true
>     openSearch: true
>     url:
>       protocol: https
>       host: url
>       port: 443
>     auth:
>       username: x
>       password: x
> ```

This means that elasticsearch is disabled but opensearch is enabled ? I'm a bit confuse

Reply:
Yes, the elasticsearch exporter is fully disabled and opensearch is enabled. I can see how the `disableExporter`  value leads to confusion.